cmd/kube-scheduler/app/server.go

```diff
@@ -140,6 +140,8 @@ func Run(ctx context.Context, cc *schedulerserverconfig.CompletedConfig, sched *
 	// To help debugging, immediately log version
 	klog.InfoS("Starting Kubernetes Scheduler", "version", version.Get())
 
```

Comment:
pre-existing, but for consistency, add version info as well?

```
klog.Infof("Version: %+v", version.Get())
```

Reply:
It's above - just in a different format. Should we change this?

Reply:
oh, missed it because the line didn't start with `version:`. nah, I wouldn't change it here